.travis.yml

```diff
@@ -5,7 +5,7 @@ matrix:
     - compiler: clang
       env:
         - CXX_COMPILER='clang++-3.9'
-        - PYTHON_VER='2.7'
+        - PYTHON_VER='3.5'
```

Comment:
You said you wanted the gcc 4.9 to be the sole py27. clang-3.9 still isn't running (it's the allowed failure). I'm going to force push this branch once I get some proof of success.